pkg/apis/internal.acorn.io/v1/imageinstance.go

```diff
@@ -34,9 +34,10 @@ type ImageInstance struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
 
-	Repo   string   `json:"repo,omitempty"`
-	Digest string   `json:"digest,omitempty"`
-	Tags   []string `json:"tags,omitempty"`
+	Project string   `json:"project,omitempty"`
```

Comment:
Open question. Is it fine to be adding another field to Image and ImageInstance? Translation between pub/priv works fine, and it is omitempty for json parsing.

Reply:
yeah, as mentioned above, putting project on the object is probably problematic in general